sockpuppet/reflex.py

```diff
@@ -36,10 +36,11 @@ def get_context_data(self, *args, **kwargs):
         view.request = self.request
         try:
             view.kwargs = resolved.kwargs
-            context = view.get_context_data()
-        except AttributeError:
             view.get(self.request)
-            context = view.get_context_data()
```

Comment:
Actually this is the way I would like it to be. 
```
context = view.get_context_data()
```

May throw an attribute error because the get request hasn't executed. (See the django list view). So if an AttributeError is thrown we execute the get part of the view as well. 

The reason why we don't do it all the time, is that I didn't want to do it unless necessary. Might shave off a couple of ms.

Reply:
A question here; are you saying that resolved may not always have an attribute `kwargs`.

Reply:
Sorry, I think I got things a bit wrong here. I was hoping not to do anything unnecessary, but it seems like everything works properly as-is. (Also had tests failing on me, but found out that was just my 2015 Macbook being too slow)

Maybe make the kwargs line not be inside the try would be better, but there's no big issue here, so I think I'll close this pull request.